time LatestResultStamp
time CurAnalyzeStamp
int32 FaceID
string name

